libvast/test/system/indexer.cpp

```diff
@@ -36,6 +36,8 @@
 using namespace caf;
 using namespace vast;
 
+/*
```

Comment:
Is this TODO or blocked by some other work?

Reply:
Sorry, these tests are obsolete, because INDEXER actors no longer have the full table index. I'll see if I can provide new, reasonable unit tests for the new INDEXER actors tomorrow.

Reply:
I've re-implemented the test.